internal/configs/virtualserver.go

```diff
@@ -343,17 +354,41 @@ func generateUpstream(upstreamName string, upstream conf_v1alpha1.Upstream, isEx
 			MaxConns:    generateIntFromPointer(upstream.MaxConns, cfgParams.MaxConns),
 			Resolve:     isExternalNameSvc,
 		}
+
+		if isPlus {
+			s.SlowStart = generateSlowStartForPlus(upstream, lbMethod)
+		}
+
 		upsServers = append(upsServers, s)
 	}
 
 	return version2.Upstream{
 		Name:      upstreamName,
 		Servers:   upsServers,
-		LBMethod:  generateLBMethod(upstream.LBMethod, cfgParams.LBMethod),
+		LBMethod:  lbMethod,
 		Keepalive: generateIntFromPointer(upstream.Keepalive, cfgParams.Keepalive),
 	}
 }
 
+func generateSlowStartForPlus(upstream conf_v1alpha1.Upstream, lbMethod string) string {
+
+	slowStart := upstream.SlowStart
+
+	if slowStart == "" {
+		return ""
+	}
+
+	for _, nLBMethod := range incompatibleLBMethodsForSlowStart {
```

Comment:
to check for the incompatible method, I suggest using approach similar to https://github.com/nginxinc/kubernetes-ingress/blob/master/internal/configs/parsing_helpers.go#L141-L148 :
1) first check if the method starts with hash
2) check if the method is in the map. This will require changing `incompatibleLBMethodsForSlowStart` to a map.